File: java-rest-api/src/main/java/com/gft/managementservice/dto/CertificateDto.java
package com.gft.managementservice.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.*;

import java.io.Serializable;
import java.util.Date;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@ToString
@JsonIgnoreProperties(ignoreUnknown = true)
public class CertificateDto implements Serializable {
    private Integer id;
    private String name;
    private String grade;
    private Float score;
    private Date expiredDate;
}

File: flutter_app/lib/src/dto/certificate_dto.dart
import 'package:fixnum/fixnum.dart' as $fixnum;

class CertificateDto {
  const CertificateDto(
      this.id, this.name, this.grade, this.score, this.expiredDate);

  final int id;
  final String name;
  final String grade;
  final double? score;
  final $fixnum.Int64? expiredDate;
}
